InAppMessaging/Example/App/InAppMessaging-Example-iOS/AutoDisplayFlowViewController.m

```diff
@@ -77,6 +78,12 @@ - (IBAction)triggerAnalyticEventTapped:(id)sender {
   [FIRAnalytics logEventWithName:self.eventNameText.text parameters:@{}];
 }
 
+- (IBAction)triggerProgrammaticallyTapped:(id)sender {
+  NSLog(@"Trigger message programmatically: %@", self.eventNameText.text);
```

Comment:
`Trigger event '$'  ` as the log here - it's not necessarily triggering a message, which is why we explicitly name the api 'triggerEvent' vs 'triggerMessage'